js/tictactoe.js

```diff
@@ -0,0 +1,229 @@
+var playerOneMove = "X";
+var playerTwoMove = "O";
+var playerOneTurn = "Player 1, please make your move."
+var playerTwoTurn = "Player 2, please make your move."
+var playerOneWinMsg = "Congratulations Player 1! You won the game!"
+var playerTwoWinMsg = "Congratulations Player 2! You won the game!"
+var defaultCellColor = "green";
+var xCellColor = "blue";
+var oCellColor = "red";
+var playerTurn = 1; //Counter to determine which character to place
+var allBoxes = document.getElementsByTagName("span");
+var gameStatus = document.getElementById("gameStatus");
+var checkCol1 = document.getElementsByClassName("col1");
+var checkCol2 = document.getElementsByClassName("col2");
+var checkCol3 = document.getElementsByClassName("col3");
+var checkRow1 = document.getElementsByClassName("winrow1");
+var checkRow2 = document.getElementsByClassName("winrow2");
+var checkRow3 = document.getElementsByClassName("winrow3");
+var checkDiag1 = document.getElementsByClassName("diag1");
+var checkDiag2 = document.getElementsByClassName("diag2");
+
+//Function to clear the board
+document.getElementById("clear").addEventListener("click", function() {
+	for (var i = 0; i < allBoxes.length; i++) {
+			allBoxes[i].innerHTML = "";
+			allBoxes[i].style.backgroundColor = defaultCellColor;
+	}
+	playerTurn = 1;
+	gameStatus.innerHTML = playerOneTurn;
+
+});
+
+//Function to check for winner. To run after each move.
+var checkForWinner = function() {
```

Comment:
Great job on a working winner function! We'll also work on ways to make these long `if/else` statements more readable and maintainable. But for tic-tac-toe, the problem is small enough to where this works fine.